src/browserWindow.ts

```diff
@@ -61,7 +62,12 @@ export class BrowserWindow {
             const ir = isRelative(f);
 
             // if it's a website, we can let the default handler deal with it
-            // by falling out of this function.
+            // by falling out of this function. Unless we're on linux, because
+            // it's special
+            if (isWeb && os.platform() === "linux") {
+              util.openWithDefaultApp(f);
+              return;
+            }
```

Comment:
this check can be within the check right below it, and we can say we need to check if xdg-open exists on linux

Reply:
I'll neaten things up.